docs/source/cpp/parquet.rst

```diff
@@ -542,6 +542,19 @@ As an example, when serializing an Arrow LargeList to Parquet:
   :func:`ArrowWriterProperties::store_schema` was enabled when writing the file;
   otherwise, it is decoded as an Arrow List.
 
+Field Id
+----------
+
+The Parquet format supports an optional integer "field id" which can be assigned
+to a field. This is used in the `iceberg specification <https://github.com/apache/iceberg/blob/main/format/spec.md#column-projection>` __
+
+On writer side, If ``PARQUET:field_id`` is present as a metadata key on a field, 
+and the corresponding value is a non-negative integer, then it will be used as 
+the "field id" in the parquet file.
```

Comment:
```suggestion
On the writer side, If ``PARQUET:field_id`` is present as a metadata key on an Arrow field, 
and the corresponding value is a non-negative integer, then it will be used as 
the "field id" in the Parquet file.
```